ui-tests/src/test/java/io/syndesis/qe/steps/other/ODataSteps.java

```diff
@@ -0,0 +1,76 @@
+package io.syndesis.qe.steps.other;
+
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import io.cucumber.datatable.DataTable;
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+import io.syndesis.qe.utils.HTTPResponse;
+import io.syndesis.qe.utils.HttpUtils;
+import io.syndesis.qe.utils.ODataUtils;
+import lombok.extern.slf4j.Slf4j;
+import okhttp3.Headers;
+import org.json.JSONObject;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Slf4j
+public class ODataSteps {
+
+    @When("^create OData credentials$")
+    public void createODataCredentials() {
+        Account oData = new Account();
+        oData.setService("OData");
+        Map<String, String> properties = new HashMap<>();
+        properties.put("serviceUri", ODataUtils.getOpenshiftService());
+        oData.setProperties(properties);
+        AccountsDirectory.getInstance().addAccount("odata", oData);
+        log.info("Created new Account: odata");
+    }
+
+    @When("^.*insert entity \"([^\"]*)\" into \"([^\"]*)\" collection on OData service$")
```

Comment:
Kind of both. initially I used it, but since I use service with prepopulated data I know, I removed the step. But as you mentioned below, it would be better to insert entities in order to manipulate with them (read, delete,...). So it will be used.